systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/BuildUtils.java

```diff
@@ -6,11 +6,13 @@
 
 import io.fabric8.openshift.api.model.BuildStatus;
 import io.strimzi.systemtest.TestConstants;
-import io.strimzi.systemtest.resources.openshift.BuildConfigResource;
 import io.strimzi.test.TestUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+import static io.strimzi.systemtest.resources.CrdResourceClients.buildConfigClient;
+import static io.strimzi.systemtest.resources.CrdResourceClients.buildsClient;
```

Comment:
Do we really want to have BuildConfig and Builds in next to our (or external project's) CRDs? From my POV it is native resource on OpenShift and this way makes it a little bit strange. Would be great to have also input from others.

Reply:
So you will remove the encapsulation or?

Reply:
I guess yes, I would keep it if it would be on more places, but in this case I would just remove it.

Reply:
Removed